docgen/templates/api.md

```diff
@@ -107,3 +72,24 @@ Value | Description
 {{end}}{{/* end enum values range */}}
 {{end}}{{/* end enum range */}}
 {{end}}{{/* end enum if */}}
+
+{{- define "message"}}
+
+Name | Type | Description
+---- | ---- | -----------
+{{range $f := .Fields}}{{$f.Name}} | {{$f.Type.Name}} |{{GetText $f.Comment.Leading}}
```

Comment:
should there be a space before `{{GetText $f.Comment.Leading}}`

Reply:
thanks, fixed!